mods/ra/weapons/explosions.yaml

```diff
@@ -245,6 +245,13 @@ V2Explode:
 	Inherits: SCUD
 	-Report:
 
+BuildingExplode:
+	Warhead@1Eff: CreateEffect
+		Explosions: building, building_napalm, large_explosion, self_destruct
```

Comment:
Because a lot of buildings didn't list that before.
I thought that might be intentional, but didn't feel like adding a second weapon for that, so I removed it from the default because we already use the 'medium' napalm.

That being said, I don't really mind adding it back.